code/drasil-example/Drasil/SWHS/Body.hs

```diff
@@ -116,7 +116,7 @@ symbMap = cdb (qw heatEInPCM : symbolsAll) -- heatEInPCM ?
   ++ map nw prodtcon ++ map nw physicCon ++ map nw mathcon ++ map nw mathcon' ++ map nw specParamValList
   ++ map nw fundamentals ++ map nw educon ++ map nw derived ++ map nw physicalcon ++ map nw unitalChuncks
   ++ [nw swhsPCM, nw algorithm] ++ map nw compcon ++ [nw materialProprty])
-  (cw heatEInPCM : map cw symbols ++ srsDomains) -- FIXME: heatEInPCM?
+  (cw heatEInPCM : map cw symbols ++ srsDomains ++ map cw specParamValList) -- FIXME: heatEInPCM?
```

Comment:
Without these concepts added to the map, we get an error for not having many reachable concepts (all auxiliary constants that weren't in the table of symbols). I'm not entirely sure which change in the code results in this change of concept enforcement, but I think it looks like a positive one since it now seems to enforce that the symbols are there.

(Also, regarding the note that I made 2 comments up, this change is required regardless of the translation.)

Reply:
It would be interesting to know a bit more as to why this is needed. I agree that it is an improvement!